<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">
    
    <node pkg="uav_detect" name="backproject_location" type="backproject_display_node" output="screen" >

      <remap from="~image_rect" to="/$(arg uav_name)/rs_d435/color/image_rect_color" />
      <remap from="~camera_info" to="/$(arg uav_name)/rs_d435/color/camera_info" />
      <remap from="~localized_uav" to="/$(arg uav_name)/uav_localization/localized_uav" />

    </node>

  </group>

</launch>
